index.js

```diff
@@ -26,9 +26,16 @@ function redbloom(initialState, options) {
 	var observable = Rx.Observable
 		.fromEvent(instance, 'dispatch')
 		.concatMap(action => {
+			var listActions = () => {
+				var listActions = bloomrun.list(action);
+				if (listActions.length == 0) {
+					listActions.push(bloomrun.lookup(action))
+				}
+				return listActions;
+			}
 			return Rx.Observable
-				.from(bloomrun.list(action))
-				.map(reducer => reducer.bind(observable, action));
+				.from(listActions())
+				.map(reducer =>	reducer.bind(observable, action));
```

Comment:
Additionally, the changes here are not necessary as it was behaving as expected which was to provide a default reducer function if none was found.

From what I could tell, it looks like these changes the behaviour by pushing the default reducer twice.